### PR TITLE
Optimize BlueR PID parameters

### PR DESCRIPTION
**Pull request by ils15**
Optimized BlueR PID settings for hotend and Bed

### CODE REVIEW

**Comment by ellensp** [comment]
Indenting in changes needs cleaned up.

**Comment by ils15** [comment]
> Indenting in changes needs cleaned up.

why? I use same indeting as old version cfg

**Comment by ellensp** [comment]
have a look at the files changed, #if starting to far over,  #define's not in a row...

**Comment by qwewer0** [comment]
Decimal point could be aligned too.

**Comment by ils15** [comment]
everything is fine now
notepad++ shows correct alignment, but i cannot see this was as "tab" instead spaces
😄

File: config/examples/Two Trees/BlueR/BlueR V1/Configuration.h [hunk 1]
```diff
@@ -492,17 +492,17 @@
   //#define PID_PARAMS_PER_HOTEND // Uses separate PID parameters for each extruder (useful for mismatched extruders)
                                   // Set/get with gcode: M301 E[extruder number, 0-2]
 
-  #if ENABLED(PID_PARAMS_PER_HOTEND)
+    #if ENABLED(PID_PARAMS_PER_HOTEND)
```
Comment (qwewer0) [inline config/examples/Two Trees/BlueR/BlueR V1/Configuration.h]: Indenting here.

File: config/examples/Two Trees/BlueR/BlueR V2/Configuration.h [hunk 2]
```diff
@@ -493,17 +493,17 @@
   //#define PID_PARAMS_PER_HOTEND // Uses separate PID parameters for each extruder (useful for mismatched extruders)
                                   // Set/get with gcode: M301 E[extruder number, 0-2]
 
-  #if ENABLED(PID_PARAMS_PER_HOTEND)
+    #if ENABLED(PID_PARAMS_PER_HOTEND)
```
Comment (qwewer0) [inline config/examples/Two Trees/BlueR/BlueR V2/Configuration.h]: Indenting here.

File: config/examples/Two Trees/BlueR/BlueR V1/Configuration.h (hunk 1 above)
Comment (ils15) [reply config/examples/Two Trees/BlueR/BlueR V1/Configuration.h]: thanks

File: config/examples/Two Trees/BlueR/BlueR V2/Configuration.h (hunk 2 above)
Comment (ils15) [reply config/examples/Two Trees/BlueR/BlueR V2/Configuration.h]: ok

**Pull request merged**